pandas/tests/groupby/test_function.py

```diff
@@ -1296,36 +1307,32 @@ def __eq__(self, other):
 # --------------------------------
 
 
-def test_size(df):
-    grouped = df.groupby(["A", "B"])
+@pytest.mark.parametrize("group_key", ["A", "B", ["A", "B"]])
+def test_size(df, group_key):
+    grouped = df.groupby(group_key)
     result = grouped.size()
     for key, group in grouped:
         assert result[key] == len(group)
 
-    grouped = df.groupby("A")
-    result = grouped.size()
-    for key, group in grouped:
-        assert result[key] == len(group)
 
-    grouped = df.groupby("B")
-    result = grouped.size()
-    for key, group in grouped:
-        assert result[key] == len(group)
+@pytest.mark.parametrize("key", ["A", "B", ["A", "B"]])
+@pytest.mark.parametrize("sort", [True, False])
```

Comment:
This brings the total number of sort parameterizations for [True, False] to 12. so maybe now worth considering creating a fixture for this. (as a follow-up)

Reply:
I think that is already a fixture for ```sort``` but it's only for ```[None, False]``` IIRC.

Reply:
we can use different fixtures with the same name. The fixture used depends on the directory structure.

we have the relevant fixture in pandas\tests\reshape\test_concat.py, so just needs to be moved.

The sort fixture in pandas\tests\indexes\conftest.py has a caution note. maybe worth adding something like that.

BUT, if you do this, do in a follow-up, not here.